packages/front-end/components/Experiment/PValueGuardrailResults.tsx

```diff
@@ -0,0 +1,174 @@
+        `alert-${status}`
+      )}
+    >
+      {status === "success" && <FaCheckCircle className="mr-1" />}
+      {status === "warning" && <FaExclamationTriangle className="mr-1" />}
+      {status === "danger" && <FaExclamation className="mr-1" />}
+      {status === "secondary" && <FaQuestionCircle className="mr-1" />}
+      <Tooltip body={<MetricTooltipBody metric={metric} />} tipPosition="right">
+        <Link href={`/metric/${metric.id}`}>
+          <a className="text-dark font-weight-bold">{metric.name}</a>
+        </Link>
+      </Tooltip>
+    </div>
+  );
+};
+
+const PValueGuardrailResults: FC<{
+  data: SnapshotVariation[];
+  variations: ExperimentReportVariation[];
+  metric: MetricInterface;
+}> = ({ data, variations, metric }) => {
+  const pValueThreshold = usePValueThreshold();
+
+  const results: PValueGuardrailResult[] = useMemo(() => {
+    return variations.map((v, i) => {
+      const stats = data[i]?.metrics?.[metric.id];
+      const expectedDirection = metric.inverse
+        ? stats.expected < 0
+        : stats.expected > 0;
+      const statSig = stats.pValue < pValueThreshold;
+      const users = data[i].users;
+      const name = v.name;
+      return {
+        stats,
+        expectedDirection,
+        statSig,
+        users,
+        name,
+        hasEnoughData: hasEnoughData(
+          stats.value,
+          data[0].metrics[metric.id]?.value
+        ),
+      };
+    });
+    // eslint-disable-next-line
+  }, [variations]);
+
+  return (
+    <div className="d-flex flex-column" key={metric.id}>
+      <HeaderResult metric={metric} results={results} />
+
+      <div>
+        <table
+          className={clsx("rounded table table-bordered experiment-compact")}
+        >
+          <thead>
+            <tr>
+              <th>Variation</th>
+              <th>Value</th>
+              <th className="text-center">P-value</th>
+            </tr>
+          </thead>
+          <tbody>
+            {results.map((r, i) => {
+              if (!r.stats) {
+                return (
+                  <tr key={i}>
+                    <td>{r.name}</td>
+                    <td colSpan={2}>
+                      <em>no data yet</em>
+                    </td>
+                  </tr>
+                );
+              }
+
+              return (
+                <tr key={i}>
+                  <td>{r.name}</td>
+
+                  <MetricValueColumn
+                    metric={metric}
+                    stats={r.stats}
+                    users={r.users}
+                  />
+
+                  {!i ? (
+                    <td></td>
+                  ) : r.hasEnoughData ? (
+                    <td
+                      className={clsx("chance result-number align-middle", {
+                        won: r.expectedDirection && r.statSig,
+                        lost: !r.expectedDirection && r.statSig,
+                        warning: !r.expectedDirection,
```

Comment:
from slack, this needs to be ` && !r.statSig` or this needs to evaluate to be mutually exclusive. right now, we add `lost warning` instead of just `lost` when something is both wrong direction and statsig.

Reply:
Good catch, addressed